ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/EeaSendRawTransaction.java

```diff
@@ -53,7 +53,7 @@ public JsonRpcResponse doResponse(final JsonRpcRequestContext requestContext) {
 
     final String enclaveKey;
     try {
-      enclaveKey = privateTransactionHandler.sendToOrion(privateTransaction);
+      enclaveKey = privateTransactionHandler.sendTransaction(privateTransaction);
```

Comment:
better than SendToOrion :)
But I suspect this whole function should PROBABLY go into the PrivateTransactionHandler, and this should just be a wrapper

Reply:
maybe it could be "submitTransaction" - i.e. remove concept of 'send'

Reply:
I like the idea of sendTransaction returning the privacyGroupId and the enclaveKey as they are used together in both sendTransaction and distributeTransaction apis.

Reply:
Unless it's including creating the marker transaction and submitting the transaction to the pool don't think it should be submitTransaction. But perhaps it should do all that? We would still need to do the send transaction and validation for the distributeTransaction RPC though.